client/txn_test.go

```diff
@@ -75,7 +75,7 @@ func TestTxnRequestTxnTimestamp(t *testing.T) {
 		{makeTS(20, 1), makeTS(20, 1)},
 	}
 
-	testIdx := 0
+	var testIdx int
```

Comment:
What's the complain here? Initializing `testIdx` in this manner seems perfectly kosher.